src/main/java/Group15/ExerciseDetailsView.java

```diff
@@ -0,0 +1,105 @@
+package Group15;
+
+import Group15.Api.Exercise;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.*;
+import javafx.geometry.Pos;
+import javafx.scene.text.Font;
+import javafx.scene.text.FontWeight;
+
+public class ExerciseDetailsView {
+
+    private static Exercise benchPress = new Exercise("Bench Press",
+            "This is a description of bench press. It is a compound exercise that works multiple muscle groups including the chest, shoulders, and triceps. Just press Bro!",
+            "Compound",
+            "Chest, Triceps, Shoulders",
+            "Barbell, Bench",
+            "Intermediate");
+
+    public static Scene createScene() {
```

Comment:
Move path to image into the exercise object, then createScene should take a Exercise as input